deploy/helm_charts/dc_website/templates/ingress.yaml

```diff
@@ -29,10 +29,9 @@ kind: Ingress
 metadata:
   name: {{ .Values.ingress.name }}
   namespace: {{ .Values.namespace.name }}
-  {{- with .Values.ingress.annotations }}
   annotations:
```

Comment:
this is good, would be nice to have more static resources like this (so it's easier to know what it is)

Reply:
Agreed, will follow this style from now on!